src/components/PlioQuestion.vue

```diff
@@ -127,6 +147,11 @@ export default {
       isAnswerCorrect: false,
       isAnswerSubmitted: false,
       showButtonLoading: false,
+      updatedProgressBarInfo: {
+        "config": this.progressBarInfo['config'],
+        "updatedCompletionPercent": 0,
+        "totalQuestions": this.progressBarInfo['totalQuestions']
+      }
```

Comment:
Everytime a question interface is injected into the main HTML, this object is created for that interface. It contains the config of the progress bar, total questions in that plio and the `updatedCompletionPercent`. This `updatedCompletionPercent` is initially set to what the completion % was before this question appeared.
`PlioQuestions.vue` receives this information as props from `Player.vue`